### PR TITLE
Display report containers as logs in the right panel

### PR DESCRIPTION
**Pull request by dbraquart**

### CODE REVIEW

File: src/utils/report-log.mapper.ts [hunk 1]
```diff
@@ -8,27 +8,30 @@
...
     const highestSeverity = mapSeverity(report.severities ?? [REPORT_SEVERITY.UNKNOWN.name]);
-    // For now, we want to display only reports that are not containers in the "log" view
+    // We display a report line in the "log" view for both
+    // - a leaf (no sub-report)
+    // - and a container (have sub-reports), if its highest severity belongs to the severity filter (or unk)
     if (
-        report.subReports.length === 0 &&
-        highestSeverity.name !== REPORT_SEVERITY.UNKNOWN.name &&
-        report.parentId !== null
+        report.parentId &&
```
Comment (TheMaskedTurtle) [inline src/utils/report-log.mapper.ts:23]: I prefer to keep explicit check on null, in JS empty string is falsy for example so `if (report.parentId)` could mean a lot of different stuff

**Review by TheMaskedTurtle — APPROVED** [review]

File: src/utils/report-log.mapper.ts (hunk 1 above)
Comment (dbraquart) [reply src/utils/report-log.mapper.ts]: I removed '!== null" on purpose to solve an issue ...

When we retrieve a report[] from the back, we build a meta-report called "Logs" having an undefined parent_id. Then we display a log "Logs" on the right side.

Now I set a null parent in the "Logs" report.

Comment (TheMaskedTurtle) [reply src/utils/report-log.mapper.ts]: oh ok, so maybe let it undefined and check report.parentId !== undefined ?

Comment (dbraquart) [reply src/utils/report-log.mapper.ts]: yes I was hesitating between the 2 options. Let's use undefined.

Comment (dbraquart) [reply src/utils/report-log.mapper.ts]: or "!= null" which works for both null and undefined

**Pull request merged**